app/lib/view/conversations/send.dart

```diff
@@ -0,0 +1,266 @@
+  }
+
+  Future<void> newConversation(String text) async {
+    final bool status = await connectionChecker.connected;
+    if (!status) {
+      showDialog(
+          context: context,
+          builder: (context) {
+            return AlertDialog(
+              icon: const Icon(Icons.wifi_off),
+              title: Text(AppLocalizations.of(context)!.noInternetConnection),
+              actions: [
+                FilledButton(
+                    onPressed: () async {
+                      Navigator.pop(context);
+                    },
+                    child: const Text("Ok")),
+              ],
+            );
+          });
+      return;
+    }
+
+    final textMessage = Message(
+      text: text,
+      own: true,
+      date: DateTime.now(),
+      author: null,
+      state: MessageState.first,
+      status: MessageStatus.sent,
+    );
+
+    final CreationResponse response = await client.conversations
+        .createConversation(
+            widget.creationData!.receivers,
+            widget.creationData!.type?.name,
+            widget.creationData!.subject,
+            text);
+
+    if (response.success) {
+      Navigator.of(context).pushReplacement(MaterialPageRoute(
+          builder: (context) => ConversationsChat(
+              title: widget.creationData!.subject,
+              id: response.id!,
+              newSettings: NewConversationSettings(
+                  firstMessage: textMessage,
+                  settings: ConversationSettings(
+                      id: response.id!,
+                      groupChat:
+                          widget.creationData!.type == ChatType.groupOnly,
+                      onlyPrivateAnswers: widget.creationData!.type ==
+                          ChatType.privateAnswerOnly,
+                      noReply:
+                          widget.creationData!.type == ChatType.noAnswerAllowed,
+                      own: true)))));
+    } else {
+      showDialog(
+          context: context,
+          builder: (context) {
+            return AlertDialog(
+              icon: const Icon(Icons.error),
+              title:
+                  Text(AppLocalizations.of(context)!.errorCreatingConversation),
+              actions: [
+                FilledButton(
+                    onPressed: () async {
+                      Navigator.pop(context);
+                    },
+                    child: const Text("Ok")),
+              ],
+            );
+          });
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          leading: IconButton(
+            onPressed: () {
+              Navigator.pop(context, null);
+            },
+            icon: const Icon(Icons.arrow_back),
+          ),
+          actions: [
+            IconButton(
+              onPressed: () {
+                _controller.clear();
+              },
+              icon: const Icon(Icons.backspace),
```

Comment:
Maybe `Icons.delete_forever` would be a better fit here? Any thoughts

Reply:
yup fits better